contrib/helloimage/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM docker://docker.io/library/alpine:latest
+
```

Comment:
Can we add a MAINTAINER section here

Reply:
MAINTAINER is deprecated, but I added a maintainer and artist label